recipes.js

```diff
@@ -9,3 +9,78 @@ mongoose.connect('mongodb://localhost/recipeApp')
     console.error('Error connecting to mongo', err)
   });
 
+
+  
+  
+  const recipeSchema = new Schema ({
+    title: {
+      type: String,
+      required: true
+    },
+    level: {
+      type: String,
+      enum: ['Easy Peasy', 'Amateur Chef', 'UltraPro Chef'],
+    },
+    ingredient: Array, 
+    cousine: {
+      type: String,
+      required: true,
+    },
+    dishType: {
+      type: String,
+      enum: ['Breakfast', 'Dish','Snack','Drink','Dessert','Other'],
+    },
+    image: {
+      type: String,
+      default: 'https://images.media-allrecipes.com/images/75131.jpg',
+    },
+    duration: {
+      type: Number,
+      min: 0,
+    },
+    creator: String,
+    create: {
+      type: Date,
+      default: Date.now,
+    }
+    
+  })
+    // We create a document in the database "recipe"
+  
+  const Recipe = mongoose.model("Recipe", recipeSchema);
+  
```

Comment:
Did you have trouble with creating one recipe (iteration 2) ? 
Here is a way to do it : 
```
Recipe.create({ 
    title: 'Too Asian Glazed Chicken Thighs',
    level: 'Amateur Chef',
    ingredients: ['1/2 cup rice vinegar', '5 tablespoons honey', '1/3 cup soy sauce (such as Silver Swan®)', '1/4 cup Asian (toasted) sesame oil', '3 tablespoons Asian chili garlic sauce', '3 tablespoons minced garlic', 'salt to taste', '8 skinless, boneless chicken thighs'],
    cousine: 'Asian',
    dishType: ['Dish'],
    image: 'https://images.media-allrecipes.com/userphotos/720x405/815964.jpg',
    duration: 40,
    creator: 'Chef LePapu'
   })
    .then(recipeDoc => {
     console.log("recipe create SUCCESS!!", recipeDoc)
   })
   .catch(err => {
     console.log("recipe create FAILURE!! 💩", err);
  });
```